File: sources/com/s8/stack/web/carbide/cube/CubeElement.java
package com.s8.stack.web.carbide.cube;

import com.s8.io.bohr.neon.core.NeBranch;
import com.s8.io.bohr.neon.core.NeObject;

public class CubeElement extends NeObject {

	
	public CubeElement(NeBranch branch, String typeName) {
		super(branch, typeName);
	}

	
	public void setZIndex(int zIndex) {
		vertex.setUInt8("zIndex", zIndex);
	}


}

File: web-sources/S8-pkgs-ui-carbide/cube/CubeElement.js

import { NeObject } from '/S8-core-bohr-neon/NeObject.js';


import { S8WebFront } from '/S8-pkgs-ui-carbide/S8WebFront.js';


/**
 * 
 */
S8WebFront.CSS_import('/S8-pkgs-ui-carbide/cube/Cube.css');

export class CubeElement extends NeObject {


    constructor(){
        super();
    }

    S8_render(){ /* continuous rendering approach... */ }

    S8_set_zIndex(level){
        this.getEnvelope().style.zIndex = level;
    }

    S8_dispose(){ /* continuous rendering approach... */ }
}